app/controllers/application_controller/ci_processing.rb

```diff
@@ -1052,7 +1052,7 @@ def process_show_list(options = {})
   end
 
   def breadcrumb_name(_model)
-    ui_lookup_for_model(self.class.model.name).pluralize
+    ui_lookup(:models => self.class.model.name)
```

Comment:
Singular / plural form is derived from the key name: `:model` vs. `:models` or `:table` vs `:tables`.